scribblings/quickscript.scrbl

```diff
@@ -295,12 +295,12 @@ There are some additional properties:
   If the script is persistent, the counter increases at each invocation of the script via the menu,
   whereas it always displays 1 if the script is not persistent.
 
-  @bold{Note:} Persistent scripts can be "unloaded" by clicking on the
-  @gui{Scripts|Manage scripts|Unload persistent scripts} menu item.
-  In the previous example, this will reset the counter. Make sure to unload a persistent script
+  @bold{Note:} Persistent scripts can be stopped by clicking on the
```

Comment:
It's not clear enough that's stopping a persistent script also resets it.
How about "can be stopped and reset"?

Reply:
done